backend/pkg/redis/redis.go

```diff
@@ -17,17 +17,28 @@ package redis
 import (
 	"context"
 	"crypto/tls"
+	"errors"
 	"fmt"
 	"net"
 	"net/url"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/redis/go-redis/v9"
 )
 
 type Client = redis.Cmdable
 
+const (
+	TemporaryValue           = "TEMP_VALUE"
```

Comment:
```suggestion
	CacheInvalidValue           = "TEMP_VALUE"
```
Would it be an option to reserve the empty string as an invalidated cache entry?
```suggestion
	CacheInvalidValue           = ""
```

Reply:
Just noticed that the empty string is an invalidated cache entry. Can we change this constant too?

Reply:
The reason why we can't use empty string is because we need to differentiate between cache miss and this temprory value. Here are the 2 scenarios :
1. We get a empty string (cache miss): we fetches the data from db then we **cache** the results .
2. We get  the temp value:  we fetches the data from db then we **dont cache** the results.

Reply:
Won't a cache miss return a `redis.Nil` error?

Reply:
You are totally correct. Still little bit rusty after coming back 🙃